packages/make-styles/src/runtime/resolveStaticStyleRules.ts

```diff
@@ -0,0 +1,28 @@
+import { expand } from 'inline-style-expand-shorthand';
+import { HASH_PREFIX } from '../constants';
+import { MakeStaticStyles, MakeStylesResolvedRule } from '../types';
+import { hashString } from './utils/hashString';
+import { compileStaticCSS } from './compileStaticCSS';
+
+export function resolveStaticStyleRules(
+  styles: MakeStaticStyles,
+  result: Record<string, MakeStylesResolvedRule> = {},
+): Record<string, MakeStylesResolvedRule> {
+  if (typeof styles === 'string') {
+    addResolvedStyles(styles, result);
+  } else {
+    const expandedStyles: MakeStaticStyles = expand(styles);
+
+    Object.keys(expandedStyles).forEach(property => {
+      const value = expandedStyles[property];
+      const staticCSS = compileStaticCSS(property, value);
+      addResolvedStyles(staticCSS, result);
+    });
```

Comment:
Do we support this use case?

```jsx
makeStaticStyles({
  body: {
    color: "blue",
    div: { color: "red" } // This guy is nested
  }
});
```

If yes - we should make use recursion there
If no - let's drop object syntax and there is no value in it

Reply:
I would say no unless there is a convincing reason/need. updated typing